move-maker.js

```diff
@@ -16,8 +16,31 @@
         ];
 */
 function validateMove(move, board) {
-    // Implement this at the end if you have time, otherwise you can help your teammates!
+  // Implement this at the end if you have time, otherwise you can help your teammates!
+  const getSplitedArray = move.split(',');
+
+  const moveIncludesComma = move.includes(',');
+  const checkIfArrayLengthTwo = getSplitedArray.length === 2;
+
+  const checkFirstValue = getSplitedArray[0] >= 1 && getSplitedArray[0] <= 3;
+  const checkSecondValue = getSplitedArray[1] >= 1 && getSplitedArray[1] <= 3;
```

Comment:
You should notice that you are using [type coercion](https://developer.mozilla.org/en-US/docs/Glossary/Type_coercion), which works and in some circumstances it is good, but you should aware of the risk: https://www.quora.com/What-are-the-dangers-of-type-coercion-in-JavaScript